rllib/env/vector_env.py

```diff
@@ -248,7 +248,16 @@ def __init__(
 
     @override(VectorEnv)
     def vector_reset(self):
-        return [e.reset() for e in self.envs]
+        # Use reset_at(index) to restart and retry until
```

Comment:
Wait, aren't we breaking the design here that the Sampler should take care of reset errors (instead of the env itself)? What if the user wants these exceptions to bubble up (and crash the worker)? Meaning what if the user has `restart_failed_sub_environments=False` (which is the default)?

Reply:
I am using the same reset_at(i) here. reset_at(i) will bubble up the Exception if ```restart_failed_sub_environments=False``` right?
I guess the point here is that there should not be difference between resetting a single sub env and resetting all the sub envs at once?